ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -829,6 +903,13 @@ boolean isInProgress() {
     }
 
     /**
+     * @return true if the blob is being decrypted
+     */
+    boolean onDecryptMode() {
```

Comment:
minor: I would rather this method be called `isDecrypting()` to match the other state check methods.

Reply:
I had a confusion between isBeingDecrypted, isDecrypting, onDecryptMode and eventually chose this. Will see if others have any take on this. If not, will switch at the end.